pkg/crc/logging/logging.go

```diff
@@ -1,14 +1,13 @@
 package logging
 
 import (
-	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
-	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 
 	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
```

Comment:
eh, no?

Reply:
@gbraad didn't get you we want to have different section for imported packages from what std library have so I just move those 2 around, nothing changed.